doc/source/changes/version_0_26.rst.inc

```diff
@@ -112,6 +112,87 @@ Miscellaneous improvements
     >>> group_even_year
     year[2000, 2002, 2004, 2006, 2008, 2010, 2012, 2014, 2016]
 
+* renamed `split_axis` as `split_axes` which now allows to split several axes at once (closes :issue:`366`):
+
+    >>> combined = ndrange('a_b = a0_b0..a1_b1; c_d = c0_d0..c1_d1')
+    >>> combined
+    a_b\c_d  c0_d0  c0_d1  c1_d0  c1_d1
+      a0_b0      0      1      2      3
+      a0_b1      4      5      6      7
+      a1_b0      8      9     10     11
+      a1_b1     12     13     14     15
+    >>> # equivalent to combined.split_axes() which split all axes
```

Comment:
I would either remove the comment here (in the release notes) because you describe that just below and/or combine the release notes of both improvements.